src/client/components/Input/Input.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Input.css';
+
+export default function Input(props) {
+  const [value, setValue] = useState();
```

Comment:
It's good to have a default for the initial state. This could be an empty string or null. Also, could the initial value be passed in as a prop?